benchmark/docker-compose.yml

```diff
@@ -1,19 +1,19 @@
-version: '3'
+version: '3.2'
 
 services:
   prometheus:
-    image: prom/prometheus:v2.30.3
+    build:
+      context: .
+      dockerfile: Dockerfile_prometheus
     extra_hosts:
       - host.docker.internal:host-gateway
-    ports:
-      - "9090:9090"
     volumes:
-      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - ${PROMETHEUS_YAML:-./prometheus.yml}:/etc/prometheus/prometheus.yml
 
   grafana:
     image: grafana/grafana:8.2.1
     ports:
-      - "3000:3000"
+      - 3000
```

Comment:
Use `ENV` variable instead. This way we need to call `docker inspect` in order to find grafana's port.